queries/orderbook/batch_rewards.sql

```diff
@@ -81,17 +81,18 @@ WITH observed_settlements AS (SELECT
                               FROM participation_data
                               GROUP BY participant),
      primary_rewards as (SELECT rpt.solver,
-                                SUM(capped_reward)  as total_reward_eth,
-                                SUM(execution_cost) as total_exececution_cost_eth
+                                SUM(capped_payment) as payment_wei,
+                                SUM(execution_cost) as exececution_cost_wei
                          FROM reward_per_auction rpt
                          GROUP BY solver),
-     aggregate_results as (SELECT concat('0x', encode(pc.solver, 'hex'))  as solver,
-                                  coalesce(total_reward_eth, 0)           as total_reward_eth,
-                                  coalesce(total_exececution_cost_eth, 0) as total_execution_cost_eth,
+     aggregate_results as (SELECT concat('0x', encode(pc.solver, 'hex')) as solver,
+                                  coalesce(payment_wei, 0)               as payment_eth,
+                                  coalesce(exececution_cost_wei, 0)      as execution_cost_eth,
                                   num_participating_batches
                            FROM participation_counts pc
                                   LEFT OUTER JOIN primary_rewards pr
                                                   ON pr.solver = pc.solver)
 
 select *
-from aggregate_results;
\ No newline at end of file
+from aggregate_results
+order by solver;
```

Comment:
Only a few field name changes in this file.

```suggestion
order by solver;

```